foia_hub/management/commands/load_agency_contacts.py

```diff
@@ -1,16 +1,9 @@
 from django.core.management.base import BaseCommand
-from foia_hub.settings.base import BASE_DIR
+from foia_hub.settings.base import BASE_DIR, DEFAULT_DATA_REPO
 from foia_hub.scripts.load_agency_contacts import process_yamls
-
-# TODO: Figure out repo structure to make this generic.
-# Things are moving around, leaving this here for now.
-
-DEFAULT_YAML_FOLDER = 'foia/contacts/data'
-
-
-def _get_yaml_folder():
-    return BASE_DIR.rstrip('/foia_hub').rstrip('foia-') + DEFAULT_YAML_FOLDER
-
+import os
+import shutil
+import subprocess
 
 class Command(BaseCommand):
```

Comment:
The help_text on this command should probably be changed to indicate that this will download the data if you want.